project-generated-code-backend/Backend/Controller/HospitalLogInController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using health_clinic_class_diagram.Backend.Model.Util;
+using health_clinic_class_diagram.Backend.Service.HospitalAccountsService;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace health_clinic_class_diagram.Backend.Controller
+{
+    public class HospitalLogInController
+    {
+        public HospitalLogInService hospitalLogInService;
+
+        public HospitalLogInController()
+        {
+            hospitalLogInService = new HospitalLogInService();
+        }
+
+        public TypeOfUser GetUserType(string jmbg, string password)
+        {
+            return hospitalLogInService.GetUserType(jmbg, password);
+        }
+    }
+}
```

Comment:
Maybe you should consider that maybe we should not change the "project-generated-code-backend" at all.
Maybe we should just make a controller in our project folder "GraphicEditorApp" . 
You don't need to change this.

Reply:
I have spoken with one of dev member he told me where exactly to create classes ,so i did.

Reply:
I think login logic will be used by other to.